socorro/external/rabbitmq/crashstorage.py

```diff
@@ -299,3 +299,29 @@ class ReprocessingRabbitMQCrashStore(RabbitMQCrashStorage):
         False
     )
 
+
+#==============================================================================
+class ReprocessingOneRabbitMQCrashStore(ReprocessingRabbitMQCrashStore):
+    required_config = Namespace()
+    required_config.rabbitmq_class = change_default(
+        RabbitMQCrashStorage,
+        'rabbitmq_class',
+        ConnectionContext,
+    )
+    required_config.routing_key = change_default(
+        RabbitMQCrashStorage,
+        'routing_key',
+        'socorro.reprocessing'
+    )
+
+    def __init__(self, config, quit_check_callback=None):
+        super(ReprocessingRabbitMQCrashStore, self).__init__(
+            config, quit_check_callback=quit_check_callback
+        )
```

Comment:
Skipping? Hmm... I don't see that. What I do see is that there's no reason for this local `__init__` to exist since it just calls the parent class without anything special.